pygotham/frontend/talks.py

```diff
@@ -100,6 +100,13 @@ def proposal(pk=None):
         db.session.add(talk)
         db.session.commit()
 
+        speaker = Speaker(
```

Comment:
A primary speaker ensures that someone that I invite as a copresenter can't come along and steal my talk away from me. A boolean field on the speaker table probably satisfies this issue (along with some check constraint that makes sure we don't end up with more than one primary speaker for a given talk).

`recording_release` should probably become a property that computes the `AND` of all of a talk's speakers' recording releases rather than be removed entirely.